File: tags/connmgmt_race_fixes_fin/TAO/performance-tests/Latency/TestImpl.java
//
// $Id$
//

public class TestImpl extends _TestImplBase {
  public TestImpl (org.omg.CORBA.ORB orb) {
    orb_ = orb;
  }
  public long test_method(long stamp) {
    return stamp;
  }
  public void shutdown () {
      // orb_.shutdown ();
  }

  private org.omg.CORBA.ORB orb_;
}

File: tags/ACE+TAO+CIAO-6_1_1/TAO/orbsvcs/tests/Bug_3444_Regression/Test_impl.cpp
// $Id$

#include "Test_impl.h"



Server_impl::Server_impl (CORBA::ORB_ptr orb)
  : orb_ (CORBA::ORB::_duplicate (orb))
{
}

void Server_impl::shutdown ()
{
  this->orb_->shutdown (0);
}
